components/global/drupal-layout.vue

```diff
@@ -3,7 +3,7 @@
     class="layout-section"
   >
     <div v-if="content">
-      <component :is="renderCustomElements(content)" />
+      <component :is="renderCustomElements(element)" v-for="(element, index) in content" :key="index" />
```

Comment:
I think this would be fixed by https://github.com/drunomics/nuxtjs-drupal-ce/pull/299/files